linuxbrew-core/Dockerfile

```diff
@@ -1,14 +1,18 @@
 FROM ubuntu
 MAINTAINER Shaun Jackman <sjackman@gmail.com>
 
+ENV DEBIAN_FRONTEND noninteractive
+
 RUN apt-get update
-RUN apt-get install -y curl g++ gawk m4 make patch ruby tcl
+RUN apt-get install -qq curl g++ gawk m4 make patch ruby tcl git
```

Comment:
git is [installed by Linuxbrew-standalone](https://github.com/sjackman/docker-linuxbrew/blob/master/linuxbrew-standalone/Dockerfile#L22). I'd rather use a brewed git.